src/kibana/components/vislib/visualizations/tile_map.js

```diff
@@ -135,20 +138,45 @@ define(function (require) {
     };
 
     /**
+     * get min and max for all cols, rows of data
+     *
+     * @method getMaxMin
+     * @param data {Object}
+     * @return {Object}
+     */
+    TileMap.prototype.getMinMax = function (data) {
+      var min = [];
+      var max = [];
+      var allData = data.rows ? data.rows : data.columns ? data.columns : [data];
```

Comment:
Multiple ternaries violate the style guide: https://github.com/elastic/kibana/blob/master/STYLEGUIDE.md#only-use-ternary-operators-for-small-simple-code